.github/workflows/auto-start-ci.yml

```diff
@@ -0,0 +1,61 @@
+---
+name: Auto Start CI
+
+on:
+  push:
+  schedule:
+    # Runs every five minutes (fastest the scheduler can run). Five minutes is
+    # optimistic, it can take longer to run.
```

Comment:
To remind our future selfs and new contributors, it would make sense to add a sentence or two about the _why_ we're not using the more traditional pull request/push events here -- like you already did in the PR description.

Reply:
I have such comment on the commit queue PR, will copy/paste it here tomorrow